### PR TITLE
Add breakpoints test cases for empty methods

### PR DESCRIPTION
**Pull request by yui-knk**
When stopped by breakpoint for an empty method
whose body is empty, the line of `def method_name`
would be displayed.

### CODE REVIEW

File: test/commands/break_test.rb [hunk 1]
```diff
@@ -108,6 +108,46 @@ def test_setting_breakpoint_to_invalid_location_shows_an_error
...
+          1:  module Byebug
+          2:    #
+          3:    # Toy class to test breakpoints
+          4:    #
+          5:    class #{example_class}
+          6:      def self.a
+          7:      end
+          8:
+          9:      def b
+         10:
```
Comment (deivid-rodriguez) [inline test/commands/break_test.rb]: Is the extra empty line intentional?

**Comment by deivid-rodriguez** [comment]
Thanks @yui-knk! Added a couple of comments, it's great to add this edge cases to the test suite!

File: test/commands/break_test.rb (hunk 1 above)
Comment (yui-knk) [reply test/commands/break_test.rb]: Yes it is intentional to test lineno 10 is not displayed.

Comment (deivid-rodriguez) [reply test/commands/break_test.rb]: :+1:

**Comment by yui-knk** [comment]
@deivid-rodriguez I added one-line method case and some comments about why `#b` has extra empty line.

**Comment by yui-knk** [comment]
❤️
